dune

```diff
@@ -1,3 +1,3 @@
 (dirs :standard \ var)
 
-(vendored_dirs ocurrent ocluster ocaml-dockerfile ocaml-version opam)
+(vendored_dirs opam)
```

Comment:
I don't see an `opam` directory anywhere in the source tree. Can we get rid of this stanza entirely?